### PR TITLE
fix: add settings context to add a tag for pagination

### PR DESCRIPTION
**Pull request by VitaliiDC8**

### CODE REVIEW

File: src/constructor/BlogConstructorProvider.tsx [hunk 1]
```diff
@@ -35,6 +38,7 @@ export const BlogConstructorProvider: React.FC<BlogConstructorProviderProps> = (
         <RouterContext.Provider value={router} key="router-context" />,
         <MobileContext.Provider value={Boolean(isMobile)} key="is-mobile-context" />,
         <DeviceContext.Provider value={device} key="device-context" />,
+        <SettingsContext.Provider value={settings} key="settings-context" />,
```
Comment (NikitaCG) [inline src/constructor/BlogConstructorProvider.tsx:41]: add info about this context in context README

File: src/components/Paginator/components/PaginatorItem.tsx [hunk 2]
```diff
@@ -20,27 +21,38 @@ export const PaginatorItem = ({
...
-                onClick={(event) => event.preventDefault()}
-            >
-                {content}
-            </a>
+            {content}
         </Button>
     );
+
+    return (
+        <>
```
Comment (NikitaCG) [inline src/components/Paginator/components/PaginatorItem.tsx:44]: we usually use `Fragment`

**Review by NikitaCG — CHANGES_REQUESTED** [review]
see in comments

File: src/components/Paginator/components/PaginatorItem.tsx (hunk 2 above)
Comment (VitaliiDC8) [reply src/components/Paginator/components/PaginatorItem.tsx:44]: ok

File: src/constructor/BlogConstructorProvider.tsx (hunk 1 above)
Comment (VitaliiDC8) [reply src/constructor/BlogConstructorProvider.tsx:41]: done

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**